src/jit/codegenarmarch.cpp

```diff
@@ -284,15 +284,21 @@ void CodeGen::genPutArgStk(GenTreePutArgStk* treeNode)
                 genConsumeAddress(addrNode);
                 addrReg = addrNode->gtRegNum;
 
+                // If addrReg equal to loReg, swap(loReg, hiReg)
+                // This reduces code complexity by only supporting one addrReg overwrite case
+                if (loReg == addrReg)
+                {
+                    loReg = hiReg;
```

Comment:
loReg & hiReg are just temporaries assigned in line 215-216 above.  At this point I do not think either have been used.